src/sql/parts/notebook/notebook.component.ts

```diff
@@ -170,6 +186,38 @@ export class NotebookComponent extends AngularDisposable implements OnInit {
 		this.notificationService.error(error);
 	}
 
+	protected initActionBar() {
+		let kernelInfoText = document.createElement('div');
+		kernelInfoText.className ='notebook-info-label';
+		kernelInfoText.innerText = 'Kernel: ';
+
+		let kernelsDropdown = new KernelsDropdown(this.contextViewService, this.modelRegistered);
+		let kernelsDropdownTemplateContainer = document.createElement('div');
+		kernelsDropdownTemplateContainer.className = 'notebook-toolbar-dropdown';
+		kernelsDropdown.render(kernelsDropdownTemplateContainer);
+		attachSelectBoxStyler(kernelsDropdown, this.themeService);
+
+		let attachToDropdown = new AttachToDropdown(this.contextViewService);
+		let attachToDropdownTemplateContainer = document.createElement('div');
+		attachToDropdownTemplateContainer.className = 'notebook-toolbar-dropdown';
+		attachToDropdown.render(attachToDropdownTemplateContainer);
+		attachSelectBoxStyler(attachToDropdown, this.themeService);
+
+		let attachToInfoText = document.createElement('div');
```

Comment:
OK, so in a separate review it'd be great to get a single component to define a "DropdownWithLabel" and handle all standard layout of it.